BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Help/DigitalInvoiceHelpViewController.swift

```diff
@@ -58,6 +58,10 @@ final class DigitalInvoiceHelpViewController: UIViewController {
         scrollView.addSubview(contentView)
         contentView.addSubview(stackView)
 
+        let backButton = GiniBarButton(ofType: .back(title: "Digitalinvoice+"))
```

Comment:
[Clarification] Why do we have the titles like "Digitalinvoice+"?

Reply:
It was jist for debugging